tests/general/e2e/controller/cookie_controller_spec.cr

```diff
@@ -30,11 +30,11 @@ describe "CookieController" do
     body["max_age"].should eq 5000
     body["http_only"].should eq false
     body["path"].should eq "/"
-    cookies = HTTP::Cookies.from_headers(response.headers)
-    cookies[cookie_name]?.as(HTTP::Cookie).value.should eq cookie_value_obj[:cookie_value]
-    cookies[cookie_name]?.as(HTTP::Cookie).path.should eq "/"
-    cookies[cookie_name]?.as(HTTP::Cookie).secure.should eq false
-    cookies[cookie_name]?.as(HTTP::Cookie).http_only.should eq false
+    cookies = response.cookies
+    cookie = cookies[cookie_name]
+    cookie.value.should eq cookie_value_obj[:cookie_value]
+    cookie.secure.should eq false
```

Comment:
not sure we should test this @ujjwalguptaofficial, this is `crystal` responsibility not `shivneri`, imho

Reply:
Its ok to test @waghanza , sometime framework does some modification and it might break expected output.